open-banking-brasil-financial-api-1_ID1.md

```diff
@@ -291,80 +277,81 @@ The authentication methods are:
 
 To performe a MFA authentication is necessary the end user to present at least two different methods as listed above. A unique method used more than once is not accepted as MFA.
```

Comment:
It is important to translate this section. Suggests:

* **LoA2**: mecanismo de autenticação com a adoção de um único fator 
* **LoA3**: Mecanismo de autenticação com múltiplos fatores de autenticação

A segunte regra deve ser adotada para o mecanismo de autenticação:

* **Para controle de acesso às API´s definidas na FASE 2 (leitura de dados)**: os `Authorization Servers` das instituições trasmissoras de dados devem condicionar a autenticação do usuário proprietário do dado,  no mínimo, a adoção de método compatível com `LoA2`. A adoção de mecanismo de autenticação mais rigoroso (`LoA3`) fica a critério da instituição transmissora de acordo com sua avaliação de riscos.
* **Para acesso às API´s das fases subsequentes (pagamento, atualização de dados etc)**: o acesso deve ser condicionado à método de autenticação compatível com `LoA3` ou superior.  

**Esclarecimentos adicionais sobre fatores de autenticação**

São fatores de autenticação:
* Aquilo que **você conhece**, como uma senha ou frase secreta
* Aquilo que **você tem**, como um token, smartcard ou dispositivo
* Aquilo que **"você é"** (algo condicionado a uma característica física exclusivamente sua), como a validação por biometria

Para realizar autenticação por múltiplos fatores (MFA) é necessário que o usuário apresente, ao menos, dois diferentes  fatores dos listados acima. Um mesmo fator usado mais de uma vez - por exemplo, a apresentação de suas senhas que ele conhece - não pode ser aceito como MFA.